ios/KMMIC/Sources/Application/App.swift

```diff
@@ -7,17 +7,20 @@
 //  Copyright © 2023 Nimble. All rights reserved.
 //
 
+import KMPNativeCoroutinesCombine
+import KMPNativeCoroutinesCore
 import Shared
 import SwiftUI
+import Combine
 
 @main
 struct KMMApp: App {
 
     @UIApplicationDelegateAdaptor private var appDelegate: AppDelegate
-
+    var subscription: AnyCancellable?
```

Comment:
What is this property?

Reply:
@suho update at [02e7eba](https://github.com/markgravity/kmm-ic/pull/69/commits/02e7eba009d5903ccdc7fb1710b65b03bafacbab) to remove the test code 🙈